sklearn/decomposition/fastica_.py

```diff
@@ -454,6 +455,9 @@ def __init__(self, n_components=None, algorithm='parallel', whiten=True,
                  fun='logcosh', fun_args=None, max_iter=200, tol=1e-4,
                  w_init=None, random_state=None):
         super(FastICA, self).__init__()
+        if max_iter < 1:
+            raise ValueError("max_iter should be greater than 1, got "
+                             "(max_iter=%d)" % max_iter)
```

Comment:
I would use `.format`, just feels simpler (some reason why:  I don't have to remember which letter does what after the `%`, `max_iter` will be truncated if it happens to be a float, weird error if `max_iter` is a tuple), but perceptions may vary depending on slowly dying habits ;-).